services/ui-src/src/components/layout/Header.test.tsx

```diff
@@ -16,44 +18,58 @@ describe("<Header />", () => {
       render(headerComponent);
     });
 
-    test("Navigation, Logo, Help and Menu is visible on Header", () => {
+    test("Logo, Help and Menu is visible on Header", () => {
       const header = screen.getByRole("navigation");
       expect(header).toBeVisible();
-      expect(screen.getByAltText("HCBS logo")).toBeVisible();
-      expect(screen.getByAltText("Help")).toBeVisible();
+      // find img elements
+      expect(screen.getByRole("img", { name: "HCBS logo" })).toHaveAttribute(
+        "alt",
+        "HCBS logo"
+      );
+      expect(screen.getByRole("img", { name: "Help" })).toHaveAttribute(
+        "alt",
+        "Help"
+      );
+      expect(screen.getByRole("img", { name: "Account" })).toHaveAttribute(
+        "alt",
+        "Account"
+      );
       expect(screen.getByAltText("Arrow down")).toBeVisible();
     });
 
-    test("Renders My Account menu and is clickable", () => {
-      render(
-        <MenuOption
-          text={"My Account"}
-          icon={"arrows/icon_arrrow_down.svg"}
-          altText={"Arrow down"}
-        />
-      );
-
-      const menuButton = screen.getByRole("button", { name: /my account/i });
+    test("When My Account menu is clicked, it expands", async () => {
+      // Find button
+      const menuButton = screen.getByRole("button", {
+        name: "my account",
+        expanded: false,
+      });
       expect(menuButton).toBeInTheDocument();
-
-      userEvent.click(menuButton);
+      // Click and expand
+      await act(async () => {
+        await userEvent.click(menuButton);
+      });
+      expect(
+        screen.getByRole("button", { name: "my account", expanded: true })
+      ).toBeInTheDocument();
     });
 
-    test("Logs out user", () => {
-      render(
-        <MenuOption
-          text={"Log Out"}
-          icon={"icon_logout.svg"}
-          altText={"Logout"}
-        />
-      );
-
-      const logoutButton = screen.getByRole("img", { name: /Logout/i });
+    test("Logs out user", async () => {
+      // Find button
+      const menuButton = screen.getByRole("button", {
+        name: "my account",
+      });
+      await act(async () => {
+        await userEvent.click(menuButton);
+      });
```

Comment:
This is a fun one! `userEvent` actually knows all about the React render cycle, so it already wraps calls in `act()` internally - we shouldn't need to worry about it at all.

Once again the Word of the Author is here: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily

If the test starts giving you warnings when `act()` is removed, we can resolve those another way.

```suggestion
      await userEvent.click(menuButton);
```